proxy/envoy/testdata/lds-httpproxy.json.golden

```diff
@@ -0,0 +1,55 @@
+{
+  "listeners": [
+   {
+    "address": "tcp://0.0.0.0:15002",
+    "name": "http_0.0.0.0_15002",
+    "filters": [
+     {
+      "type": "read",
+      "name": "http_connection_manager",
+      "config": {
+       "codec_type": "auto",
+       "stat_prefix": "http",
+       "generate_request_id": true,
+       "tracing": {
+        "operation_name": "ingress"
```

Comment:
When we have ingress and egress listeners, this becomes egress for egress listener